### ISSUE TITLE
Update settings for retention policy together with plugin settings

### ISSUE THREAD

**Issue by terma** [jenkinsci/ec2-fleet-plugin#111]
```IdleRetentionStrategy``` has reference on cloud, because Jenkins create new instance of cloud each settings modification, existent nodes uses old policy.

After #43 investigation.

**Comment by terma** [comment]
Fix released under version [1.9.2](https://github.com/jenkinsci/ec2-fleet-plugin/releases/tag/ec2-fleet-1.9.2)

**Comment by ianfixes** [comment]
Does this apply to changes to the fleet name?  If so, I think there might be a regression.  I'm using 1.16.1.  Should I open a new issue?

**Comment by terma** [comment]
@ianfixes could you please add details about problem, what happens? when it happens? How often? Some precondition. Thx. So I can try to reproduce and fix

**Comment by ianfixes** [comment]
OK, I'll plan to open a new issue if/when I experience this problem again.  No action necessary for now.
